keyboards/polykybd/tools/status_oled_preview.py

```
@@ -121,36 +259,112 @@ def draw_bitmap(setpix, data, ox, oy, w=16, h=16):
+        draw(setp, tiny, max(x, nx), base, txt)
+
+
+def build_panel(side, disp, small, icons, tiny, brightness=50, rgb=(128, 255, 100, 80, 5, 'Rainbow'),
+                lang='en-US', wpm=0):
     """side: 'L' (USB host, layout panel) or 'R' (bridge, RGB panel). The role word
     (USB/Link) follows is_usb_host_side() on hardware; here 'L' models the USB half.
+    rgb=None models RGB being switched off, which re-flows BOTH panels.
     Returns set of (x,y) pixels."""
+    rgb_on = rgb is not None
     pts = set()
     setp = lambda px, py: pts.add((px, py))
```

Ruff (0.16.0)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `setp` as a `def`

(E731)